src/coreclr/src/vm/dllimport.cpp

```diff
@@ -6308,7 +6308,7 @@ class LoadLibErrorTracker
 
 // Load the library directly. On Unix systems, don't register it yet with PAL.
 // * External callers like System.Runtime.InteropServices.NativeLibrary.Load() need the raw system handle
-// * Internal callers like LoadLibraryModule() can convert this handle to a HMODULE via PAL APIs on Unix
+// * Internal callers like LoadNativeLibrary() can convert this handle to a HMODULE via PAL APIs on Unix
```

Comment:
Nit: This comment is not valid anymore. We got rid of the need to register 3rd party native libraries with PAL. The PAL API to convert `NATIVE_LIBRARY_HANDLE` to `HMODULE` does not exist anymore.